databricks_sql_connector_core/tests/e2e/common/predicates.py

```diff
@@ -8,9 +8,12 @@
 
 
 def pysql_supports_arrow():
-    """Import databricks.sql and test whether Cursor has fetchall_arrow."""
-    from databricks.sql.client import Cursor
-    return hasattr(Cursor, 'fetchall_arrow')
+    """Checks if the pyarrow library is installed or not"""
+    try:
+        __import__("pyarrow")
```

Comment:
`__import__` is not meant to be used by programmers. Please use regular `import` instead

Reply:
fixed